src/js/util.js

```diff
@@ -125,6 +158,8 @@ exports.isObject = isObject;
 exports.isFunction = isFunction;
 exports.isBuffer = isBuffer;
 exports.isArray = Array.isArray;
+exports.exceptionWithHostPort = exceptionWithHostPort;
+exports.errnoException = errnoException;
```

Comment:
I think `errnoException` can be adjusted to any API using `uv` internally and `exceptionWithHostPort` to  `net` module, too. Moreover, I refered to `nodev7.0`